src/main/java/wiresegal/wob/arcanum/Parsing.kt

```diff
@@ -167,6 +170,65 @@ fun about(message: Message) {
     })
 }
 
+val progressCache = mutableListOf<Pair<String, String>>()
+var progressCacheTimeStamp: Long? = null
+
+fun showProgressBar(message: Message) {
+    val projects = getProgressBar()
+    var progresses = ""
+    for (project in projects) {
+        val (name, percent) = project
+        progresses += "$name is at $percent%\n"
+    }
+
+    message.channel.sendMessage(EmbedBuilder().apply {
+        setTitle("Progress Bars")
+        setColor(embedColor)
+
+        setDescription(progresses)
```

Comment:
Rather than manually building up a textual list of all progresses, we can use embed fields to display the progress for individual projects. See the `embedFromContent` function for one instance of this. We can also turn the project names into a link if they are one on Brandon's sight (like the Starsight Release), then. This would have to be tested though, as I don't know whether field titles support named links in the Markdown.

 Another fun thing to do might be to add Unicode block elements to give a rough approximation of the percentage as a real "progress bar", for, say, 5% increments.

This could end up looking something like this:
![Embed Example](https://i.imgur.com/PNR371Q.png)

Reply:
This would look a lot nicer, I'll see if I can figure it out

Reply:
Yeah, so it seems that you can only have named hyperlinks in the actual text, not the header, not sure how you want to fix this, but one solution is to hyperlink the percentage, thought that looks slightly strange